templates/app/webpack-default/template/tiapp.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ti:app xmlns:ti="http://ti.appcelerator.org">
+	<id>__PROJECT_ID__</id>
+	<name>__PROJECT_NAME__</name>
+	<version>__PROJECT_VERSION__</version>
+	<publisher>not specified</publisher>
+	<url>not specified</url>
+	<description></description>
+	<copyright>not specified</copyright>
+	<icon>appicon.png</icon>
+	<fullscreen>false</fullscreen>
+	<navbar-hidden>false</navbar-hidden>
+	<analytics>true</analytics>
+	<guid></guid>
+	<property name="ti.ui.defaultunit">dp</property>
+	<property name="run-on-main-thread" type="bool">true</property>
```

Comment:
Let's remove the "run-on-main-thread" property.
It's no longer supported and is currently ignored.